developers/weaviate/current/retriever-vectorizer-modules/multi2vec-clip.md

```diff
@@ -45,7 +45,7 @@ services:
     - '8080'
     - --scheme
     - http
-    image: semitechnologies/weaviate:1.9.0
+    image: semitechnologies/weaviate:1.13.2
```

Comment:
@byronvoorbach – you can always make Jekyll use the latest version. So, changing `1.13.2` with `{{ site.weaviate_version }}` solves this